docs/device_support_guide.md

```diff
@@ -4,7 +4,30 @@ There are two main methods to get your MSI laptop supported: the recommended met
 > [!IMPORTANT]
 > If there are any BIOS/firmware updates available for your laptop, follow this guide for your current firmware before installing any of them. Then repeat the process for each new firmware version you install from the official MSI website. This is required to obtain support for older firmware as the EC configuration may vary across the versions.
 
-## Windows method (recommended):
+# Table of contents (rough)
+
++ FW naming and WMI versioning explanation(?) (TODO)
```

Comment:
this is outside the scope of this guide, you've already done a good [write up](https://github.com/BeardOverflow/msi-ec/discussions/98) on that topic, so lets keep it outside this guide

Reply:
Right, this is just a guide on how to obtain and interpret the dump

Reply:
Alright, adding it here as a draft wouldn't hurt, we'll see how well it fits here when the guide is near finished